statsmodels/datasets/tests/test_utils.py

```diff
@@ -19,22 +19,28 @@ def test_get_rdataset():
         assert_(duncan.from_cache)
 
 # temporarily disabled because some test servers don't have internet
-def t_est_webuse():
+def testwebuse():
     # test copied and adjusted from iolib/tests/test_foreign
-    from statsmodels.iolib.tests.results.macrodata import macrodata_result as res2
-    #base_gh = "http://github.com/statsmodels/statsmodels/raw/master/statsmodels/datasets/macrodata/"
-    base_gh = "http://statsmodels.sourceforge.net/devel/_static/"
-    res1 = webuse('macrodata', baseurl=base_gh, as_df=False)
-    assert_array_equal(res1 == res2, True)
+    # Check if internet is available
+    if check_internet() is False:
```

Comment:
Could you just use 
```
internet_availble = check_internet()
@skipf(internet_availble)
``` 
to decorate the function.  Would allow the test to be unmodified.